Sources/ScrollingTabController.swift

```diff
@@ -121,6 +121,7 @@ public class ScrollingTabController: UIViewController, UIScrollViewDelegate, UIC
     override public func viewDidAppear(animated: Bool) {
         super.viewDidAppear(animated)
         tabView.panToPercentage(scrolledPercentage)
+        self.tabView.collectionView.collectionViewLayout.invalidateLayout()
```

Comment:
This may need to be before panToPercentage, as I believe panToPercentage assumes that the collection view has already laid itself out